tutorials/polkadot-sdk/build-a-parachain/acquire-a-testnet-slot.md

```diff
@@ -0,0 +1,265 @@
+To perform any action on Rococo, you need ROC tokens, and to store the tokens, you must have access to a Substrate-compatible digital currency wallet. You can't use [development keys and accounts](https://docs.substrate.io/reference/command-line-tools/subkey/#predefined-accounts-and-keys){target=\_blank} for operations in any public setting. Many options are available for holding digital currency—including hardware wallets and browser-based applications—and some are more reputable than others. You should do your own research before selecting one.
+
+However, you can use the [Polkadot.Js Apps](https://polkadot.js.org/apps/){target=\_blank} interface to get you started for testing purposes.
+
+To prepare an account, follow these steps:
+
+1. Open the [Polkadot.Js Apps](https://polkadot.js.org/apps/){target=\_blank} interface and connect to the Rococo network
+
+    ![](/images/tutorials/polkadot-sdk/build-a-parachain/acquire-a-testnet-slot/acquire-a-testnet-slot-1.webp)
+
+2. Navigate to the **Accounts** section
+    1. Click on the **Accounts** tab in the top menu
+    2. Select the **Accounts** option from the dropdown menu
+   
+    ![](/images/tutorials/polkadot-sdk/build-a-parachain/acquire-a-testnet-slot/acquire-a-testnet-slot-2.webp)
+
+3. Copy the address of the account you want to use for the parachain deployment
+
+    ![](/images/tutorials/polkadot-sdk/build-a-parachain/acquire-a-testnet-slot/acquire-a-testnet-slot-3.webp)
+
+4. Visit the [Polkadot Faucet](https://faucet.polkadot.io){target=\_blank} and paste the copied address in the input field. Ensure that the network is set to Rococo and click on the **Get some ROCs** button
+
+    ![](/images/tutorials/polkadot-sdk/build-a-parachain/acquire-a-testnet-slot/acquire-a-testnet-slot-4.webp)
+
+    After a few seconds, you will receive 100 ROC tokens in your account.
+
+    !!! note
+        Alternatively, you can join the [Rococo Element Channel](https://matrix.to/#/#rococo-faucet:matrix.org){target=\_blank} and send a message with `!drip` and the public address for your Rococo to get 100 ROC in your wallet. For example, send a message similar to the following: `!drip 5HErbKmL5JmUKDVsH1aGyXTGZb4i9iaNsFhSgkNDr8qp2Dvj`
+
+## Reserve a Parachain Identifier
+
+You must reserve a parachain identifier before you can register as a parathread on Rococo. The steps are similar to the ones you followed in [Connect a Local Parachain](/tutorials/polkadot-sdk/build-a-parachain/connect-a-parachain/){target=_\blank} to reserve an identifier on the local relay chain. However, for the public TestNet, you'll be assigned the next available identifier.
+
+To reserve a parachain identifier, follow these steps:
+
+1. Navigate to the **Parachains** section
+    1. Click on the **Network** tab in the top menu
+    2. Select the **Parachains** option from the dropdown menu
+
+    ![](/images/tutorials/polkadot-sdk/build-a-parachain/acquire-a-testnet-slot/acquire-a-testnet-slot-5.webp)
+
+2. Register a parathread
```

Comment:
I don't think `Register a parathread` is accurate here. Also you have a section below with the same name. It should be Register a Parachain Id or something like that